docs/blog/2023/04-01-release-v14/index.md

```diff
@@ -0,0 +1,198 @@
+---
+title: Atuin v14 - Deletions, NuShell, Windows and more!
+description: Release notes for Atuin v14! 
+slug: release-v14
+authors: [ellie]
+tags: [release]
+---
+
+Announcing a new release of Atuin! v14 is out now. Atuin allows you to easily search and sync your shell history across many machines. Get started [here](https://atuin.sh/docs/)
+
+You can update your installation via your system package manager, or by downloading the latest release from the [release page](https://github.com/ellie/atuin/releases).
+
+This release introduces some breaking changes, so please ensure all Atuin clients and servers are updated or sync may not complete.
+
+We had a lot of changes in this release - I'll call out a few, but this is not exhaustive. Read the changelog below for the full list!
+
+### Community
+
+- [Discord](https://discord.gg/Fq8bJSKPHh)
+- [Mastodon](https://hachyderm.io/@atuin)
+- [Twitter](https://twitter.com/atuinsh)
+
+## New Features
+
+### Deletions
+
+You can now delete history! This has been our longest-standing issue, and one we are frequently asked about. So it's great to get it out! It took us a while due to the nature of sync, and we wanted to try several different approaches. There is still no way to delete history from the UI, however you can now pass the `--delete` flag to any `atuin search` command to delete all history matching the query. I suggest you run the search _without_ the delete flag first, just to check what you are removing
+
+For example, to delete all commands that start with `psql`:
+
+```
+atuin search --delete --search-mode prefix psql
+```
+
+### NuShell
+
+We now support NuShell! This was a huge effort by [@stevenxxiu](https://github.com/stevenxxiu), and involved work on both Atuin and NuShell itself. The effort was greatly appreciated, thank you so much 💖
+
+Run in *Nushell*:
+
+```
+mkdir ~/.local/share/atuin/
+atuin init nu | save ~/.local/share/atuin/init.nu
+```
+
+Add to `config.nu`:
+
+```
+source ~/.local/share/atuin/init.nu
+```
+
+PRs:
+- https://github.com/nushell/nushell/pull/8560
+- https://github.com/nushell/nushell/pull/8207
+- https://github.com/ellie/atuin/pull/788
+
+### Inline history UI
+
+![inline history](inline.png)
+
+Another of our biggest asks - the UI can now be configured so that it does not take the entire screen. Thank you to [@pdecat](https://github.com/pdecat)
```

Comment:
I believe @conradludgate contributed heavily to this, with all the crossterm and tui work. Worth mentioning IMO 🙂

Reply:
I get enough credit :D